docker/nginx/default.conf

```diff
@@ -32,7 +32,7 @@ server {
     client_max_body_size 100m;
 
     # Specify what happens when PHP files are requested
-    location ~ \.php$ {
+    location ~ \.php$|\.html {
```

Comment:
Now here, I see what you're after. I don't adhere to mixing PHP in HTML anymore. But I appreciate what your use-case is.